📝 Add requested permissions (#492)
actions/stale: README.md

- Close the stale issues and pull requests after 7 days of inactivity
- If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restart

## Recommended Permissions

> For the execution of the action, it must be able to read all issues and pull requests. In addition, the action must have the permission to write to issues and/or pull requests. This can be achieved with the following [configuration in the action](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions) if the permissions are restricted:
> 
> ```yml
> permissions:
>   issues: write
>   pull-requests: write
> ```
> 
> You can find more detailed information about the required permissions under the corresponding options.

Reviewer: ```suggestion
For the execution of this action, it must be able to fetch all issues and pull requests from your repository.  
This can be achieved with the following [configuration in the action](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions) if the permissions are restricted:

yml ⚠️ >>> wrap this as code it's just an issue with the suggestion format
permissions:
  issues: write
  pull-requests: write

In addition, based on the provided configuration, the action could require more permission(s) (e.g.: add label, remove label, comment, close, etc).  
You can find more information about the required permissions under the corresponding options that you wish to use.
```

Author: Thanks for the suggestion! 😄

- Close the stale issues and pull requests after 7 days of inactivity
- If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restart

> ## Recommended Permissions
> 
> For the execution of this action, it must be able to fetch all issues and pull requests from your repository.  

Reviewer: ```suggestion
## Recommended permissions

For the execution of this action, it must be able to fetch all the issues and pull requests from your repository.  
```

This can be achieved with the following [configuration in the action](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions) if the permissions are restricted:

```yaml
permissions:
  issues: read
  pull-requests: read
```
  
In addition, based on the provided configuration, the action could require more permission(s) (e.g.: add label, remove label, comment, close, etc).  

> You can find more information about the required permissions under the corresponding options that you wish to use.

Reviewer: ```suggestion
You can find more information about the required permissions under the corresponding options that you wish to use.  
However, if don't want to bother, you can use set these permissions:

yaml
permissions:
  issues: write
  pull-requests: write

```
And fix the code syntax ;)

- Close the stale issues and pull requests after 7 days of inactivity
- If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restart

## Recommended permissions

For the execution of this action, it must be able to fetch all issues and pull requests from your repository.  
This can be achieved with the following [configuration in the action](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions) if the permissions are restricted:

```yaml
permissions:
  issues: read
  pull-requests: read
```
  
In addition, based on the provided configuration, the action could require more permission(s) (e.g.: add label, remove label, comment, close, etc.).  
You can find more information about the required permissions under the corresponding options that you wish to use.  

> However, if don't want to bother, you can use set these permissions:

Reviewer: ```suggestion
However, if don't want to bother, you can use these permissions:
```
Sorry, I suggested with a typo 😆

> Default value: unset
> 
> Required Permission: `issues: write`

Reviewer: ```suggestion
Default value: unset
Required Permission: `issues: write`
```
I think removing this extra row would make it more readable.
It's a minor change, but I would rather have something a little more compact! WDYT?

Author: But that makes it all on one line?!

Reviewer: Not if you use the double space at the end of the row

Default value: unset

> Required Permission:`pull-requests: write`

Reviewer: ```suggestion
Required Permission: `pull-requests: write`
```
